mne/io/otb/otb.py

```diff
@@ -0,0 +1,251 @@
+# Authors: The MNE-Python contributors.
+# License: BSD-3-Clause
+# Copyright the MNE-Python contributors.
+
+import tarfile
+from collections import Counter
+from datetime import datetime
+from pathlib import Path
+
+import numpy as np
+from defusedxml import ElementTree as ET
+
+from ..._fiff.meas_info import create_info
+from ...utils import _check_fname, fill_doc, logger, verbose, warn
+from ..base import BaseRaw
+
+OTB_PLUS_DTYPE = np.int16  # the file I have is int16; TBD if *all* OTB+ are like that
+FORMAT_MAPPING = dict(
+    d="double",
+    f="single",
+    i="int",
+    h="short",
+)
+OTB_PLUS_FORMAT = FORMAT_MAPPING[OTB_PLUS_DTYPE().dtype.char]
+
+
+def _parse_patient_xml(tree):
+    """Convert an ElementTree to a dict."""
+
+    def _parse_sex(sex):
+        # TODO English-centric; any value of "sex" not starting with "m" or "f" will get
+        # classed as "unknown". TBD if non-English-like values in the XML are possible
+        # (e.g. maybe the recording GUI is localized but the XML values are not?)
```

Comment:
@OTBioelettronica is "sex" always "M" or "F" in these files or can users input whatever they like in this field? If left unspecified, will the `<sex>` node just be missing from the XML file?